static/app/components/teamSelector.tsx

```diff
@@ -377,7 +377,7 @@ function TeamSelector(props: Props) {
     }
     // We only want to do this once when the component is finished loading for teams and mounted.
     // If the user decides they do not want the default, we should not add the default value back.
-  }, [fetching, useTeamDefaultIfOnlyOne]);
+  }, [fetching, useTeamDefaultIfOnlyOne]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I don't get it, why is this an error in CI all of a sudden?

Reply:
Not sure, CI was grumpy over here: https://sentry.slack.com/archives/C8V02RHC7/p1710193569187799

Reply:
This statement is missing dependencies so it should complain

Reply:
Yes but we have the warning silenced in CI do we not?

Reply:
https://github.com/getsentry/sentry/blob/e7683c892593b163eb35c81c6f073598dc0dfebd/.eslintrc.js#L10-L15

Reply:
oh thats true... but this is the last one that popped up when i was manually trying to findout what happened in that chat thread... so maybe we should enable the rule on CI so we dont' add more exhaustive-deps problems going forward! i didn't think of that before, but it'll be good

Reply:
Oh nice, seems like we fixed all of them already.

I wonder if that's true in getsentry as well.

Let's remove this from the eslint config and move it into our config repo?